test/unit/app/workers/test-calculate-workload-points.js

```diff
@@ -111,7 +111,7 @@ describe('services/workers/calculate-workload-points', function () {
 
   it('calls calculateSdrConversionPoints', function (done) {
     getAppReductions.resolves(REDUCTION_HOURS)
-    getAppCmsReductionHours.resolves(0)
+    getCmsAdjustmentPoints.resolves(0)
```

Comment:
In order to test: `var totalPoints = (workloadPoints + sdrConversionPoints + sdrPoints + paromsPoints + cmsAdjustments)` we should have this as a value other than 0 and ensure the total points value is updated appropriately, both positively and negatively.